ansible/roles/buildmaster/templates/xbps-clean-repos.sh.j2

```diff
@@ -1,71 +1,93 @@
 #!/bin/bash
 
 MAINREPO="/hostdir/binpkgs"
-REPOS="$MAINREPO $MAINREPO/nonfree $MAINREPO/debug"
-REPOS64="$MAINREPO/multilib $MAINREPO/multilib/nonfree"
-ARCHS="i686 x86_64 armv6l armv7l"
+REPOS_GLIBC="$MAINREPO $MAINREPO/nonfree $MAINREPO/multilib $MAINREPO/multilib/nonfree $MAINREPO/debug"
```

Comment:
Keeping just the ARCHS variable is preferable, it can be templated using Jinja2 from Ansible.  You could make the token `{{ xbps_clean_archs }}` and I will put a commit on top of this to populate it with the right values.

Reply:
hmm but we need to differentiate which repos are for a specific machine.

Reply:
do you mean something like: `ARCHS="$ARCHS_GLIBC $ARCHS_MUSL $ARCHS_AARCH64"` ?

Reply:
no I mean literally `ARCHS="{{ xbps_clean_archs }}"` which will be set at the time this file is written out by Ansible.

Reply:
ah ok